akalisten/templates/lotsude/style.css

```diff
@@ -217,10 +223,42 @@ footer {
 .maybe h2 { color: #f39c12; border-color: #f39c12; }
 .highlight { border-color: #f39c12; }
 
+.lucide-icon {
+    color: var(--icon);
+    width: 1em;
+    height: 1em;
+}
+.lucide-icon-15 {
+    color: var(--icon);
+    width: 1.5em;
+    height: 1.5em;
+}
+.lucide-icon-2 {
+    color: var(--icon);
+    width: 2em;
+    height: 2em;
+}
```

Comment:
Exactly, my intention was to allow for different ratios between icon size and font size. `.lucide-icon-15` shows icons that have 1.5 the font height and `.lucide-icon-2` shows icons that have twice the height of the font.
Currently, both `.lucide-icon` (1em) and `.lucide-icon-15` (1.5em) are used. I'll remove `.lucide-icon-2`.

Reply:
Then it is a purely aesthetic reason that the icons are sometimes significantly larger than the text.
With the `.multi-column-list` I also find the sizes and spacing suitable because it is enclosed by a box and can pass as an icon in the box.
While the subheading and subsequent link have the same icon size and the icon/text combinations in the footer are very prominent. For both cases I would prefer `width: 1.25em; height: 1.25em;` with `gap: 0.5ex;`.

Reply:
I have implemented my suggestions to this point. I hope that's okay. Otherwise please simply roll back the corresponding commits.